lib/services/notifications.js

```diff
@@ -12,9 +12,10 @@ const _ = require('lodash'),
  * @param {object} data
  * @returns {Promise}
  */
-function callWebhook(event, url, data) {
+function callWebhook(event, url, data, uri) {
   const headers = {
-      'X-Event': event
+      'X-Event': event,
+      'page-uri': uri
```

Comment:
The reason I shied away from Link is because it looks like it's mostly used for pagination purposes? But I don't see why we couldn't use it for this. I agree that we should generalize it more, though.